semopenalex-app/data/templates/http%3A%2F%2Fwww.metaphactory.semopenalex.com%2Fapp%2FAbout.html

```diff
@@ -177,11 +221,11 @@
 SELECT ?paperTitle ?citedByCount ?firstAuthorName WHERE {
 
   ?paper dcterms:title ?paperTitle .
-  ?paper soa:hasConcept ?Concept .
+  &lt;&lt;?paper soa:hasConcept ?Concept&gt;&gt; soa:score ?score .
   ?Concept skos:prefLabel "Semantic Web"^^xsd:string . 
   ?paper soa:citedByCount ?citedByCount .
   ?paper soa:hasAuthorPosition ?authorPosition . 
```

Comment:
we do not have Author Position class any more. we renamed it to `Authorship`. correct query, 

````
 ?paper soa:hasAuthorship ?authorship . 
  ?authorship soa:position "0"^^xsd:integer . 
  ?authorship soa:hasAuthor ?firstAuthor . 

```